less/mixins.less

```diff
@@ -1 +1,48 @@
-// Mixins in here
\ No newline at end of file
+// Mixins in here
+.link-reset(@color) {
+  text-decoration: none;
+  list-style-type: none;
+  color: @color;
+}
+
+@desktop: ~"(min-width: 800px)";
+
+.flexy(
+  @disp: flex,
+  @dir: null,
+  @wrap: null,
+  @just: null,
+	@ai: null) {
```

Comment:
While `null` is indeed used with Sass, Less does not use this value. Given that your intention seems to be to set nulled properties to use default values, this won't actually affect the appearances of your elements, but it will add a property setting that will be reported by the browser as invalid (this can be noticed when inspecting an elements using default value(s) from this mixin).